config/jobs/kubernetes-sigs/cluster-api/cluster-api-periodics-release-1-5.yaml

```
@@ -104,16 +106,19 @@ periodics:
         resources:
           requests:
             cpu: 7300m
+            memory: 9Gi
+          limits:
+            cpu: 7300m
+            memory: 9Gi
```

```suggestion
            memory: 32Gi
          limits:
            cpu: 7300m
            memory: 32Gi
```
Let's please use the same values as on release-1.4